tests/Unit/Functions/Options/TestExcludeDeferJS.php

```diff
@@ -26,6 +26,7 @@ public function testShouldReturnExcludeDeferJSArray() {
             'www.uplaunch.com',
             'google.com/recaptcha',
             'widget.reviews.co.uk',
+            'verify.authorize.net/anetseal',
```

Comment:
@crystinutzaa This array of deferred scripts, i.e. `$exclude_defer_js` is repeated in the next test. Let's pull these out of each test and move them into `setUp()` and a property such as `$this->expected_deferred_js`. Then in the next test, add the new deferred scripts to it.

Why? It'll help us as we add more deferred scripts in the future, i.e. one place to make changes instead of 2.

Reply:
I moved it in a function which returns the $exclude_defer array with excluded jquery files based on function param:
`public function get_exclude_defer_js_list( $defer_jquery ) {`